src/SFA.DAS.EmployerAccounts.Web/Orchestrators/OrganisationOrchestrator.cs

```diff
@@ -229,58 +225,58 @@ public virtual void CreateCookieData(HttpContextBase context, EmployerAccountDat
             }
         }
 
-        private static string BuildAddressString(GetEmployerInformationResponse response)
-        {
-            var addressBuilder = new StringBuilder();
-
-            if (!string.IsNullOrEmpty(response.AddressLine1))
-            {
-                addressBuilder.Append($"{response.AddressLine1}, ");
-            }
-
-            if (!string.IsNullOrEmpty(response.AddressLine2))
-            {
-                addressBuilder.Append($"{response.AddressLine2}, ");
-            }
-
-            if (!string.IsNullOrEmpty(response.TownOrCity))
-            {
-                addressBuilder.Append($"{response.TownOrCity}, ");
-            }
-
-            if (!string.IsNullOrEmpty(response.County))
-            {
-                addressBuilder.Append(string.IsNullOrEmpty(response.AddressPostcode)
-                    ? $"{response.County}"
-                    : $"{response.County}, ");
-            }
-
-            if (!string.IsNullOrEmpty(response.AddressPostcode))
-            {
-                addressBuilder.Append(response.AddressPostcode);
-            }
-
-            return addressBuilder.ToString();
-        }
-
-        public async Task<OrchestratorResponse<OrganisationDetailsViewModel>> CheckLegalEntityIsNotAddedToAccount(string hashedLegalEntityId, string userIdClaim, string legalEntityCode, OrganisationType organisationType)
-        {
-            var accountLegalEntitiesHelper = new AccountLegalEntitiesHelper(_mediator);
-            var accountEntities = await accountLegalEntitiesHelper.GetAccountLegalEntities(hashedLegalEntityId, userIdClaim);
-
-            if (accountLegalEntitiesHelper.IsLegalEntityAlreadyAddedToAccount(accountEntities, null, legalEntityCode, organisationType))
-            {
-                var conflictResponse = new OrchestratorResponse<OrganisationDetailsViewModel>
-                {
-                    Data = new OrganisationDetailsViewModel(),
-                    Status = HttpStatusCode.Conflict
-                };
-
-                return conflictResponse;
-            }
-
-            return null;
-        }
+        //private static string BuildAddressString(GetEmployerInformationResponse response)
```

Comment:
Is this meant to be commented out?

Reply:
Yeah this was left over from Davids changes I sort of left it in to help him out but realized just now its all done now so I will taken it and some other stuff out.